example/sync/main.go

```diff
@@ -57,9 +57,12 @@ func run() (err error) {
 		}
 		switch choice {
 		case 1:
-			if err := connector.Sync(); err != nil {
+			replicated, err := connector.Sync()
+			if err != nil {
 				return err
 			}
+
+			fmt.Println("%d frames synced", replicated.FrameNo)
```

Comment:
Shouldn't this be replicated.FrameSynced?

Reply:
yes good catch